obsidian.css

```diff
@@ -10,11 +10,11 @@
   --color-primary-900: #15161f;
   --color-primary-1000: #0a0b0f;
   --color-primary-1100: #050608;
-  --color-title-h1: #95ff6fc3;
```

Comment:
Colors should not be changed - in Editor one wants to identify which title hierarchy one is in. On the other hand in preview mode one wants to read and first at all wants to identify it is a title (which is identifiable through its font-size - as like users are used to through reading blogs) - to much color in Preview-Mode leads to missing focus on important elements, which should be highlighted e.g. by color.

Reply:
I know the changes where made on title colors are significant - this shell not happen anymore.